### ISSUE TITLE
"request has been deprecated" warning when installing noblox.js

### ISSUE THREAD

**Issue by Dreamurrrr** [noblox/noblox.js#143]
**Describe the issue**
When installing noblox.js, this following warning pops up in the install logs:
```
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

**To Reproduce**
1. Install noblox.js
2. See warning

**Expected behavior**
A clean install without any warnings.

**Why this Issue is Important**
A future update of npm may break the install sequence for noblox.js.

**Comment by suufi** [comment]
I'm currently working on a v5 rewrite that doesn't use request. This one will take a while.

**Comment by itwithlyam** [comment]
Usually this isn't a problem, the code should still work, as long as you don't get `npm err`, then the code will still work.
It might be a problem in the future though.

**Comment by bobthedev90368** [comment]
This usually refers to when you install older version I recommend to use: npm install noblox.js
instead of downloading a specific version. F.Y.I. downloading the first version can cause some cookie issues as the cookie endpoint has changed. Since you are installing ver: 2 it will give you a deprecated warning as ver: 3 is (I believe) the newest. It is trying to tell you that noblox may not work properly.

**Comment by FourAstro** [comment]
If it works, it works.

**Comment by royce-mathew** [comment]
Why doesn't this noblox switch to using axios? It is a much more advanced and a more up to date library

**Comment by Neztore** [comment]
Still looking for help on this to migrate to using the built in Node.js Fetch API.

**Comment by Axcelers** [comment]
I suggest we should migrate to fetch(). The API connections keeps getting left open, making noblox.js an unreliable source. My bots have been unreliable due to noblox.js using requests.

For example, once the bots start, the API will work without issue. However, due to recent issues with the request package since its been deperacated, connections have been being left open causing some noblox.js functions to not complete from time to time. For example, NBX.promote() or NBX.setRank().

This is an URGENT issue that needs to be resolved as there isn't really any other reliable package for my case.

**Comment by Axcelers** [comment]
> Still looking for help on this to migrate to using the built in Node.js Fetch API.

What parts of the noblox.js package use request? Is it only the http.js file in /lib/util?

**Comment by Sqikerz** [comment]
Why not use postman-request instead, at least temporarily? It's actively maintained and an upgrade from the 4 year old deprecated dependency that's currently used.

**Comment by Neztore** [comment]
Resolved by #810
